src/calculateRentalCost.js

```diff
@@ -29,6 +29,20 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const oneDayRentCost = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+  const longTerm = 7;
+  const shortTerm = 3;
+
+  switch (true) {
+    case days >= longTerm:
+      return days * oneDayRentCost - bigDiscount;
```

Comment:
In our case, the price calculation logic is the same, so it is better to create a variable. (Lesson 'Transportation on Vacation Solution' , 5:10) 
```suggestion
      const basePrice = days * oneDayRentCost;
      return basePrice - bigDiscount;
```